### ISSUE TITLE
Trying to establish Marinetti Uthernet connetion on Windows 10 causes crash

### ISSUE THREAD

**Issue by roughana** [digarok/gsplus-legacy-archive#82]
After configuring ethernet to be on (Uthernet slot 3) in GSPlus config and in TCPIP control panel, I attempt to make a connection from the TCPIP control panel (Marinetti 3.0 b9 with Uthernet 1.01 LL) and the GSPlus process goes away.

**Comment by salfter** [comment]
Same problem here...had to shift-boot the image, inactivate the Uthernet link layer, and reboot to get it working again.  I suspect that if I reactivate the link layer, it'll crash again.  

Tested with the latest versions of Marinetti and Speccie's Uthernet link layer on a System 6.0.4 image pulled from Asimov.

**Comment by vavruska** [comment]
I know this is and old report but I am using it just fine with the latest version of Merinetti.

**Comment by michael75431** [comment]
I see this happen running GSPlus on MacOS as soon as I hit "connect to network" after the configuration is created.

**Comment by Rowlison** [comment]
I am having the same issue with the most recent version of GSPlus and Marinetti 3.0b11 running on an otherwise stock system 6.0.4. I've enabled uthernet in GSPlus and tried starting connection (trying both Uthernet and UthernetII LinkLayers) both with slot 3 in the IIgs set to "Your Card" and the built-in 80-column setting. The GSPlus process just terminates.

**Comment by Rowlison** [comment]
> I know this is and old report but I am using it just fine with the latest version of Merinetti.

You have this working with 3.0b11? Which linklayer are you set to use and what settings?

**Comment by vavruska** [comment]
Do you have winpcap Installed? I believe this is needed to get it to work.

https://www.winpcap.org/

I have have multiple ethernet devices so I needed to select #2 but that is just my setup.

**Comment by Rowlison** [comment]
I have Win10pcap installed. I didn’t see any way to configure Win10pcap, so I was assuming it’s just a matter of installing it. GSPlus terminates when attempting to create a connection.

**Comment by vavruska** [comment]
Yes, there is no configuration for winpcap.  Perhaps try running gsplus as administrator.

**Comment by Rowlison** [comment]
I’d tried that as well. Same result.

**Comment by vavruska** [comment]
Are you setup for DHCP or static? If DHCP try a static IP...might not be getting an address.
Do you have Connect to network when GS Starts? try disabling it.
Make sure the LAN slot is 3 in the TCPIP control panel
Make sure Uthernet is choosen, not UthernetII

After that I think I am out of ideas

**Comment by michael75431** [comment]
I've made sure I'm following those  configuration options and the problem is exactly the same.

There are other TCP/IP and GSPlus settings
- GSPLus -> Ethernet Card Configuration -> "Use Interface Number" : 0

TCPIP
- DNS - using what my router tells me
- Username/password - nothing
- IP address - I put in the address of the host computer I'm on plus 1, its not used on the network
- Netmask - 255.255.255.0
- Gateway - router 10.0.1.1 (correct for my setup)
- MAC address -- auto filled with 162, no idea what that is

**Comment by Rowlison** [comment]
Same here. My understanding is that the MAC address (which is autofilled) is just setting the last octet which is arbitrary and the first octets are hard-coded into GSPlus. I have all settings as set, though my gateway is different (again, because of specific network) - and when you go to start connection, GSPlus just dies. Win10 x64.

**Comment by xlar54** [comment]
Same issue here (although I never tried installing winpcap.  Would be really nice to have this working for some new networking code development

**Comment by troydm** [comment]
Same issue, using latest Marinneti 3.0b11 and Uthernet link layer on Windows 10, I have Win10Pcap installed, GSplus v0.14 crashes immediatelly :(
